sql/src/main/java/org/apache/druid/sql/http/SqlResource.java

```diff
@@ -81,16 +85,21 @@ public Response doPost(
       @Context final HttpServletRequest req
   ) throws IOException
   {
-    final PlannerResult plannerResult;
-    final DateTimeZone timeZone;
+    final SqlLifecycle lifecycle = sqlLifecycleFactory.factorize();
+    final String sqlQueryId = lifecycle.initialize(sqlQuery);
+    final String remoteAddr = req.getRemoteAddr();
+    final String currThreadName = Thread.currentThread().getName();
 
-    try (final DruidPlanner planner = plannerFactory.createPlanner(sqlQuery.getContext())) {
-      plannerResult = planner.plan(sqlQuery.getQuery(), req);
-      timeZone = planner.getPlannerContext().getTimeZone();
+    try {
+      Thread.currentThread()
+            .setName(StringUtils.format("sql[%s]", sqlQueryId));
```

Comment:
I think this might be able to fit on one 120-col line, which if so, would be preferable.

Reply:
👌, changed into one line